sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/RefreshTokenScopesIT.java

```diff
@@ -0,0 +1,88 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.test.aad.converter;
+
+import com.azure.test.oauth.SeleniumTestUtils;
+import com.azure.test.utils.AppRunner;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.annotation.RegisteredOAuth2AuthorizedClient;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.*;
+
+public class RefreshTokenScopesIT {
+
+    @Test
+    public void testRefreshTokenConverter() {
+        try (AppRunner app = new AppRunner(DumbApp.class)) {
+            SeleniumTestUtils.addProperty(app);
+            app.property("azure.activedirectory.authorization.office.scopes", "https://manage.office.com/ActivityFeed.Read");
+            app.property("azure.activedirectory.authorization.graph.scopes", "https://graph.microsoft.com/User.Read");
```

Comment:
Hi, @lzc-1997-abel , please create another PR to set multiple scopes for graph and office. For both test and sample.
![image](https://user-images.githubusercontent.com/13167207/102970210-0f29a180-4532-11eb-9bab-c278ff40be37.png)